doc/examples/filters/plot_tophat.py

```diff
@@ -0,0 +1,31 @@
+"""
+================================================================
+Removing small objects in grayscale images with a top hat filter
```

Comment:
small -> large? Based on the wikipedia example?

Reply:
the example shows how to keep large object only by removing the top-hat image. I don't think this is important...